api/swagger.yaml

```diff
@@ -2096,6 +2096,12 @@ definitions:
       ForceUpdate:
         description: "A counter that triggers an update even if no relevant parameters have been changed."
         type: "integer"
+      Runtime:
+        description: "Runtime is the type of runtime specified for the task executor."
+        type: "string"
+      RuntimeData:
+        description: "RuntimeData is the payload sent to be used with the runtime for the executor."
```

Comment:
do we want to expose this to the user? Rather than a well-formed object (however docker decides to do so)?